### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by ricardobranco777**
More Pylint & Flake8 fixes

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,10 +13,10 @@ static-bash:
 	shellcheck destroy.sh -o all -s bash -S info
 
 static-pylint:
-	@find scripts -type f -not -path "scripts/qesap/.tox/*" -not -path "scripts/qesap/.venv/*" -name \*.py -exec pylint --rcfile=scripts/qesap/pylint.rc qesap.py {} +
+	@find scripts * -type f -not -path "scripts/qesap/.tox/*" -not -path "scripts/qesap/.venv/*" -name \*.py -exec pylint --rcfile=scripts/qesap/pylint.rc qesap.py {} +
 
 static-flake8:
-	@find scripts -type f -not -path "scripts/qesap/.tox/*" -not -path "scripts/qesap/.venv/*" -name \*.py -exec flake8 --ignore=E501 {} +
+	@find scripts * -type f -not -path "scripts/qesap/.tox/*" -not -path "scripts/qesap/.venv/*" -name \*.py -exec flake8 --ignore=E501 {} +
```
Comment (mpagot) [inline Makefile:19]: how it works with this star?

File: scripts/qesap/qesap.py [hunk 2]
```diff
@@ -359,7 +359,7 @@ def cmd_ansible(configure_data, base_project, dryrun, verbose, destroy=False):
 
     for command in ansible_cmd_seq:
         if dryrun:
-            print(' '.join(command['cmd']))
+            print(' '.join(command))
```
Comment (mpagot) [inline scripts/qesap/qesap.py:362]: Thanks, was a bug

File: scripts/qesap/pylint.rc [hunk 3]
```diff
@@ -161,7 +161,8 @@ disable=line-too-long,
         unused-argument,
         too-many-locals,
         too-many-arguments,
-        duplicate-code
+        duplicate-code,
+	invalid-name
```
Comment (mpagot) [inline scripts/qesap/pylint.rc:165]: What was the warning to suppress?

File: Makefile (hunk 1 above)
Comment (ricardobranco777) [reply Makefile:19]: > how it works with this star?

Looks in every subdirectory :)

File: scripts/qesap/pylint.rc (hunk 3 above)
Comment (ricardobranco777) [reply scripts/qesap/pylint.rc:165]: > What was the warning to suppress?

The scripts in `ansible` subdirectory.  But I would fix this only if there's the will to fix those scripts.

Comment (alvarocarvajald) [reply scripts/qesap/pylint.rc:165]: Isn't it better than to ignore all `invalid-name` errors to simply not check the ansible sub-directory?

**Review by alvarocarvajald — APPROVED** [review]
Only one question, otherwise LGTM

File: scripts/qesap/pylint.rc (hunk 3 above)
Comment (ricardobranco777) [reply scripts/qesap/pylint.rc]: > Isn't it better than to ignore all `invalid-name` errors to simply not check the ansible sub-directory?

Dropped the whole commit.

**Review by mpagot — APPROVED** [review]
LGTM

**Pull request merged**